lib/utils/types.ts

```diff
@@ -0,0 +1,31 @@
+import type { TSESLint } from '@typescript-eslint/experimental-utils';
+
+type RecommendedConfig<TOptions extends readonly unknown[]> =
+  | TSESLint.RuleMetaDataDocs['recommended']
+  | [TSESLint.RuleMetaDataDocs['recommended'], ...TOptions];
+
+// These 2 types are copied from @typescript-eslint/experimental-utils' CreateRuleMeta
+// and modified to our needs
+type TestingLibraryRuleMetaDocs<TOptions extends readonly unknown[]> = Omit<
+  TSESLint.RuleMetaDataDocs,
+  'recommended' | 'url'
+> & {
+  recommendedConfig: Record<
```

Comment:
Could you add a jsdoc to this property like [`@typescript-eslint/experimental-utils` does when typing `RuleMetaDataDocs`](https://github.com/typescript-eslint/typescript-eslint/blob/1c1b572c3000d72cfe665b7afbada0ec415e7855/packages/experimental-utils/src/ts-eslint/Rule.ts#L26) please?